design/comments.go

```diff
@@ -36,6 +36,9 @@ var commentAttributes = a.Type("CommentAttributes", func() {
 	a.Attribute("body", d.String, "The comment body", func() {
 		a.Example("This is really interesting")
 	})
+	a.Attribute("body.rendered", d.String, "The comment body rendered in HTML", func() {
```

Comment:
Does it make sense to to add a comment like `(only exists in comment responses and will be ignored during creation)`?